plugins/entity-feedback-backend/src/service/router.ts

```diff
@@ -226,6 +226,33 @@ export async function createRouter(
     res.json(responses.filter(r => accessibleEntityRefs.includes(r.userRef)));
   });
 
+  router.get('/app-ratings', async (_, res) => {
+    const ratings = await dbHandler.getAppRatings();
+    res.status(200).json(ratings);
+  });
```

Comment:
I wonder if we should anonymize this response? 😅 if someone really wants to look up based on users they got that info in the DB. Perhaps even a separate endpoint that returns aggregates as this could actually get quite large?